Fix/168620 www.m2o.it , fix/168622 www.deejay.it , update/152345 www.capital.it

superlex opened the pull request:
## Prerequisites

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

## What problem does the pull request fix?

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [x] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?

fix https://github.com/AdguardTeam/AdguardFilters/issues/168620
fix https://github.com/AdguardTeam/AdguardFilters/issues/168622
update https://github.com/AdguardTeam/AdguardFilters/issues/152345

### Enter the issue address

`https://www.m2o.it/on-air/`
`https://www.deejay.it/on-air/`
`https://www.capital.it/on-air/`

### Notes

- Added `m2o.it` and `deejay.it` to the existing `capital.it#@##iubenda-cs-banner`: if you don't reject cookies you can't listen to the webradio, so you can't hide it;

- Added `deejay.it` and `capital.it` to the existing (trusted-) scriptlets to automatically reject cookies;
- Removed `capital.it` from `#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')`: you don't need to accept cookies, you can reject them;

- Replaced `capital.it#%#//scriptlet('prevent-element-src-loading', 'script', 'static.chartbeat.com/js/chartbeat_brightcove_plugin.js')` with `redirect=noopjs` so uBO users can take advantage of it;
- Added `googletagmanager.com/gtm.js` exception needed to listen to the webradio.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

@Alex-302

superlex committed:
Update cookies_allowlist.txt - fix/168620 fix/168622

superlex committed:
Update cookies_specific.txt - fix/168622 - upd/152345

superlex committed:
Update allowlist.txt - fix/168620 fix/168622 upd/152345

Alex-302 commented on AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:
```diff
@@ -555,8 +555,10 @@ goteborg2023.com#@#.cookie-popup:not(body):not(html)
+! https://github.com/AdguardTeam/AdguardFilters/issues/168622
 ! https://github.com/AdguardTeam/AdguardFilters/issues/152345
-capital.it#@##iubenda-cs-banner
+m2o.it,deejay.it,capital.it#@##iubenda-cs-banner
```
I removed `###iubenda-cs-banner` and exclusions. We will use the domain specific rule.

Alex-302 commented on AnnoyancesFilter/Cookies/sections/cookies_specific.txt:
```diff
@@ -11667,7 +11667,7 @@ matkahuolto.fi#%#//scriptlet('trusted-set-cookie', 'CookieConsentV2', 'YES,TOI,A
 character.ai#%#//scriptlet('set-local-storage-item', 'VISITED_0', 'true')
 ilmattino.it,leggo.it,3bmeteo.com#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible button.iubenda-cs-close-btn')
-leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,capital.it,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')
+leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')
```
it can be moved above with a comment `! Required for some sites` to avoid adding a domain to a rule that doesn't need to be applied to some site.

superlex replied:
What about move 
`leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')`
 under `! iubenda CMP` section?

Then, is 
`video.repubblica.it#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button.iubenda-cs-accept-btn'); if(el) el.click(); }, 5E3); });`
needed despite
`repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')`
?

As concerns
`ilmattino.it,leggo.it,3bmeteo.com#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible button.iubenda-cs-close-btn')`
`leggo.it` and `3bmeteo.com` can be added to
`m2o.it,deejay.it,capital.it,moto.it,automoto.it,atlantico.fr,movieplayer.it,mediaset.it,libero.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-customize-btn')`
`m2o.it,deejay.it,capital.it,moto.it,automoto.it,atlantico.fr,movieplayer.it,mediaset.it,libero.it#%#//scriptlet('trusted-click-element', '#iubFooterBtn')`
while `ilmattino.it` requires more attention (also with the current filter)

Alex-302 replied:
> What about move

Ok.

> Then, is

Yep)

> As concerns

Yes. But for the sites where more attention required, add a comment

superlex replied:
Done :)
Is it now ok?

Alex-302 replied:
Yes, thanks)

Alex-302 committed:
Merge branch 'master' into fix/168620-fix/168622-update/152345

superlex committed:
Update cookies_specific.txt

Moved
```
ilmattino.it,leggo.it,3bmeteo.com#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible button.iubenda-cs-close-btn')
leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,capital.it,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')
```
under `! iubenda CMP` section

Alex-302 committed:
Merge branch 'master' into fix/168620-fix/168622-update/152345
